Kudu.SiteManagement/SiteManager.cs

```diff
@@ -154,7 +152,8 @@ public async Task<Site> CreateSiteAsync(string applicationName)
                     await OperationManager.AttemptAsync(() => WaitForSiteAsync(serviceUrls.First()));
 
                     // Set initial ScmType state to LocalGit
-                    var settings = new RemoteDeploymentSettingsManager(serviceUrls.First() + "api/settings");
+                    var credentials = _context.Configuration.BasicAuthCredential.GetCredentials();
+                    var settings = new RemoteDeploymentSettingsManager(serviceUrls.First() + "api/settings", credentials);
```

Comment:
This changes the behavior as we will now always pass basic auth regardless of appSettings existence.  Though, I can't think how it would regress any.   

@balag0  could you merge this in?  Might need rebasing.